src/components/shared/form-components/email-input/email-input.jsx

```diff
@@ -23,7 +23,7 @@ export default function EmailInput({ classNames, register, errors }) {
     <div
       className={clsx(
         styles.formGroup,
-        classNames.email,
+        classNames && classNames.email ? classNames.email : '',
```

Comment:
Something to think about: In many components, when we pass in classnames to a component, it is a string.
For example:
```
<ExampleComponent classNames={styles.customClass} />
```
 You are not introduced a new patter (which is very cool), where classNames is actually an object.
For example:
```
<ExampleComponent classNames={{ listClass: styles.ListClass }} />
```
Even though they have the same purpose, the implementation is actually different. Maybe consider a different prop name so that it is more clear when classNames expects an object, so that it is clear to us in the future.